compiler/ir/backend.common/src/org/jetbrains/kotlin/ir/builders/declarations/declarationBuilders.kt

```diff
@@ -126,6 +131,12 @@ fun IrFunctionBuilder.buildConstructor(): IrConstructor {
     }
 }
 
+inline fun buildFunWithDescriptor(originalDescriptor: FunctionDescriptor, b: IrFunctionBuilder.() -> Unit): IrFunctionImpl =
```

Comment:
Please rename 'b' to something meaningful: builder?

Reply:
I added another commit to do this refactoring in the whole file, since it seems inconsistent to only use a different name in this one function.